__init__.py

```diff
@@ -475,7 +475,7 @@ def on_gui_page_show(self, message):
                 # Check if the show_page deactivates a previous idle override
                 # This is only possible if the page is from the same skill
                 self.log.info('Cancelling idle override')
-                if (override_idle is False and
+                if (not override_idle and
```

Comment:
I believe this differentiates between `False` and `None` so should be `is False`